core/controllers/profile.py

```diff
@@ -251,7 +253,8 @@ class SignupPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         return_url = str(self.request.get('return_url', self.request.uri))
-
+        if re.match('^/[^//]', return_url) is None:
```

Comment:
Add a line comment above this line that explains what it's checking for, for the benefit of devs reading this code (to give them context on what this check is looking for and what its aim is). So they don't remove it in the future.

Reply:
done